src/main/scala/com/request/network/lib/wrappers/Web3Wrapper.scala

```diff
@@ -50,9 +51,9 @@ class Web3Wrapper(web3jService: Option[Web3jService], networkId: Option[Int])
 
   def setUpOptions(options: Any): Any = ???
 
-  def getTransactionReceipt(hash: String): Future[Any] = ???
+  def getTransactionReceipt(hash: String): CompletableFuture[EthGetTransactionReceipt] = web3j.ethGetTransactionReceipt(hash).sendAsync()
```

Comment:
we should stick to Scala `Future`

Reply:
The only reason I returned a completableFuture here, is because we didn't have the Converter in scope at his point. But if we add the FutureConverter, it would be easy to just convert it directly there.